ivy/stateful/layers.py

```diff
@@ -2224,3 +2224,58 @@ def _forward(self, x):
             The input array as it is.
         """
         return x
+
+
+class IFFT(Module):
+    def __init__(
+        self,
+        dim,
+        /,
+        *,
+        norm="backward",
+        n=None,
+        out=None,
+        device=None,
+        dtype=None,
+    ):
+        """
+        Class for applying IFFT to input.
+
+        Parameters
+        ----------
+        dim : int
+            Dimension along which to take the IFFT.
+        norm : str
+            Normalization mode. Default: 'backward'
+        n : int
```

Comment:
This details of this argument could be expanded by referring [here](https://github.com/unifyai/ivy/blob/1865ed37f143f731913fcb6ac03fe23fb90c989e/ivy/data_classes/array/experimental/layers.py#L618).

Reply:
Thanks @bipinKrishnan ..have made the changes